src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      result += number;
+    },
+
+    subtract(result, number) {
+      result -= number;
+    },
+
+    multiply(result, number) {
+      result *= number;
+    },
+
+    divide(result, number) {
+      result /= number;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, number) {
+      callback(this.result, number);
```

Comment:
The 'operate' method should pass 'this.result' as the first argument to the callback function to ensure the 'result' property is correctly updated.